docs/src/components/ThemeExample.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link, View } from '@aws-amplify/ui-react';
```

Comment:
This file was converted from docs/src/components/ThemeExample.mdx because

1. {props.component} is hard to lint in mdx
2. we don't have other mdx component in docs/